files/__addonLocation__/rollup.config.mjs

```diff
@@ -1,4 +1,4 @@
-<% if (typescript) { %>import typescript from 'rollup-plugin-ts';<% } else { %>import babel from '@rollup/plugin-babel';<% } %>
+<% if (typescript) { %>import typescript from 'rollup-plugin-ts';<% } else { %>import { babel } from '@rollup/plugin-babel';<% } %>
```

Comment:
Was this a real bug, or why this change?

Reply:
even though the package exports both default and named babel, some tooling requires the named export. 

I forget now, but I'd rather go with what's stable for all environments, than using `default` "just cause".
_I should really have written this down when I encountered it_ :(